src/lib/features/client-feature-toggles/client-feature-toggle-service.ts

```diff
@@ -44,7 +42,7 @@ export class ClientFeatureToggleService {
     async getClientDelta(
         revisionId: number | undefined,
         query: IFeatureToggleQuery,
-    ): Promise<RevisionDeltaEntry | undefined> {
+    ): Promise<ClientFeaturesDeltaSchema | undefined> {
```

Comment:
Fix the types. RevisionDeltaEntry has removed as complex objects, but we are returning string array.